assets/schema/ensemble_schema.json

```diff
@@ -808,6 +808,38 @@
         }
       }
     },
+    "LoadingContainer-payload": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "A unique identifier for this widget"
+        },
+        "isLoading": {
+          "type": "boolean",
+          "description": "Whether to display loading widget"
```

Comment:
Let's highlight that this should be an expression/string instead of a static boolean.